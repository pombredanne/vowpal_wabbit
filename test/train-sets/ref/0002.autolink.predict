0.000000 PFF/20091028
0.509756 WIP/20091028
0.136403 GCC/20091028
0.985786 AAXJ/20091028
0.507160 VWO/20091028
0.000000 EEV/20091028
0.233830 GDX/20091028
0.418682 RTH/20091028
0.634232 MXI/20091028
0.475285 EWU/20091028
0.000000 SH/20091028
0.744745 EDC/20091028
0.000000 ERY/20091028
0.044066 SDS/20091028
0.149500 OEF/20091028
0.357515 IYT/20091028
0.301717 BIL/20091028
0.044749 GLL/20091028
0.028216 EDZ/20091028
0.475252 IWM/20091028
0.497705 VXF/20091028
0.512975 IJJ/20091028
0.505257 PIN/20091028
0.530784 XLB/20091028
0.442881 ECH/20091028
0.565508 TYH/20091028
0.596249 VAW/20091028
0.557028 DBP/20091028
0.652740 XME/20091028
0.647889 VO/20091028
0.587051 RSX/20091028
0.615271 EWC/20091028
0.623903 TUR/20091028
0.512773 VYM/20091028
0.516087 FCG/20091028
0.523472 VGT/20091028
0.546759 EWQ/20091028
0.541007 IEV/20091028
0.522297 XLK/20091028
0.549899 EFG/20091028
0.551172 BKF/20091028
0.610711 KIE/20091028
0.573640 EEB/20091028
0.604650 IJK/20091028
0.130562 DUG/20091028
0.160398 TWM/20091028
0.397021 MDY/20091028
0.465489 ACWI/20091028
0.306046 BSV/20091028
0.451804 DDM/20091028
0.562548 DIA/20091028
0.421780 TLT/20091028
0.236994 DXD/20091028
0.586882 XHB/20091028
0.496520 VDE/20091028
0.485167 BND/20091028
0.607898 EMB/20091028
0.566469 SCO/20091028
0.493898 AMJ/20091028
0.350759 OIL/20091028
0.571708 PZA/20091028
0.580106 VGK/20091028
0.598505 RWX/20091028
0.472618 JJA/20091028
0.613411 FXD/20091028
0.548215 XES/20091028
0.578537 VIG/20091028
0.286715 DZZ/20091028
0.636131 VFH/20091028
0.445309 DTO/20091028
0.589898 EWP/20091028
0.583204 FDN/20091028
0.562114 INP/20091028
0.251517 TYP/20091028
0.581881 RWR/20091028
0.590790 KBE/20091028
0.225561 EUO/20091028
0.511503 IWF/20091028
0.374570 SMN/20091028
0.569382 SMH/20091028
0.559602 XRT/20091028
0.397406 USO/20091028
0.497436 DJP/20091028
0.363715 CFT/20091028
0.257487 SRS/20091028
0.568894 MOO/20091028
0.441081 BIV/20091028
0.332550 VXX/20091028
0.609218 IYM/20091028
0.545205 IFN/20091028
0.615542 SLV/20091028
0.557245 TAO/20091028
0.609056 PGF/20091028
0.605234 IYR/20091028
0.260975 QID/20091028
0.575455 THD/20091028
0.570998 IJS/20091028
0.568091 VB/20091028
0.463948 EDV/20091028
0.510333 IEZ/20091028
