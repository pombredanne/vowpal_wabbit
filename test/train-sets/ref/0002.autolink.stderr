predictions = 0002.autolink.predict
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile = train-sets/0002.dat
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
0.271591 0.271591            1            1.0   0.5211   0.0000       15
0.136121 0.000650            2            2.0   0.5353   0.5098       15
0.150865 0.165610            4            4.0   0.5854   0.9858       15
0.118837 0.086808            8            8.0   0.5575   0.4187       15
0.101632 0.084426           16           16.0   0.5878   0.3575       15
0.055032 0.008432           32           32.0   0.6038   0.6153       15
0.033363 0.011695           64           64.0   0.5683   0.4726       15

finished run
number of examples per pass = 100
passes used = 1
weighted example sum = 100.000000
weighted label sum = 52.222146
average loss = 0.023252
best constant = 0.522221
total feature number = 1499
